internal/logger/logger.go

```diff
@@ -75,7 +76,8 @@ var matchingFuncNames = [...]string{
 var (
 	quietFlag, jsonFlag, anonFlag bool
 	// Custom function to format error
-	errorFmtFunc func(string, error, bool) string
+	// can be registered by RegisterError
+	errorFmtFunc = config.FmtError
```

Comment:
Can't you make this work without adding an import of config?

It will annoying that config will not be able to import "logging".

Reply:
Perfect!